deploy/charts/harvester/dependency_charts/snapshot-validation-webhook/templates/webhook.yaml

```diff
@@ -0,0 +1,68 @@
+---
+{{ $ca := genCA "snapshot-validation-webhook-ca" 365 }}

```

Comment:
let's define it as 3650, otherwise certificate rotation will quickly become a problem.